modules/apps/commerce/commerce-product-test/src/testFunctional/tests/CommerceProductDetails.testcase

```diff
@@ -1546,6 +1546,162 @@ definition {
 		}
 	}
 
+	@description = "This is a test for COMMERCE-11650. User can checkout a product in a channel that has also stock in another channel, and the product's stock in another channel cannot be affected by this channel"
+	@priority = 5
+	test CanCheckoutAProductInAChannelWithoutAffectingAnotherChannelStock {
+		property portal.acceptance = "true";
+
+		task ("Given a Minium site with a buyer user") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(siteName = "Minium");
+
+			CommerceEntry.addAccountEntryUser(
+				accountName = "Commerce Account",
+				accountType = "Business",
+				agreeToTermsAndAnswerReminderQuery = "true",
+				createNewAccount = "true",
+				requireReset = "false",
+				userEmailAddress = "buyer@liferay.com",
+				userFirstName = "Buyer",
+				userLastName = "User",
+				userRole = "Buyer",
+				userScreenName = "buyeruser",
+				userSiteMembership = "Minium");
+		}
+
+		task ("And a second Minium site is created and buyer user is associated with it") {
+			CommerceAccelerators.initializeNewSiteViaAccelerator(
+				siteName = "Minium 2",
+				siteTemplateName = "Minium");
+
+			JSONUser.addUserToSite(
+				groupName = "Minium 2",
+				userEmailAddress = "buyer@liferay.com");
+		}
+
+		task ("And an Italy warehouse eligible for Minium Portal and a Southwest warehouse eligible for Minium 2 Portal are set") {
+			CommerceNavigator.goToWarehousesAdmin();
+
+			LexiconEntry.gotoEntryMenuItemNoError(
+				menuItem = "Delete",
+				rowEntry = "United States - Southwest");
+
+			LexiconEntry.viewNoEntry(rowEntry = "United States - Southwest");
+
+			CommerceEntry.removeEntryEligibility(
+				channelEligibility = "Minium 2 Portal",
+				entryName = "Italy");
+
+			CommerceNavigator.goToWarehousesAdmin();
+
+			CommerceEntry.removeEntryEligibility(
+				channelEligibility = "Minium Portal",
+				entryName = "United States - Northeast");
+		}
+
+		task ("And a product Brake Pads' Low Stock Threshold is set to 1 and Allow Back Orders is enabled") {
+			CommerceJSONProductsAPI._patchCommerceProductConfiguration(
+				allowBackOrder = "true",
+				productName = "Brake Pads");
+
+			CommerceProducts.openProductsAdmin();
+
+			CommerceNavigator.searchEntry(entryName = '''"Brake Pads"''');
```

Comment:
You can use this macro to search and navigate to product entry:

CommerceProducts.searchAndGoToProductEntry(entryProductName = "Brake Pads");